scout/server/blueprints/alignviewers/views.py

```diff
@@ -27,6 +30,30 @@
 LOG = logging.getLogger(__name__)
 
 
+@alignviewers_bp.route("/remote/cors/<path:remote_url>", methods=["OPTIONS", "GET"])
+def remote_cors(remote_url):
```

Comment:
Yes! It works very nicely:

![image](https://user-images.githubusercontent.com/28093618/106759414-06e57b80-6633-11eb-85ce-e0719c9dc41f.png)

Reply:
This should also be testable, but when we have time!

Reply:
Good call, will do!

Reply:
Acknowledging that is, but yeah, testable, only a little involved. Monkey patching comes to mind, but 84 likes on stack overflow must mean something.. 😉

Reply:
It basically becomes a file hosted on our server. I could try to write a test if you want, and if it takes too much time we just merge and that's it

Reply:
Yes, exactly: one could either fake its env to control that it does exactly the right thing, or just give it a url that will always be up and see that it gets a response..

Reply:
I was thinking google

Reply:
It crossed my mind.. 😁